online-inference/stable-diffusion/service/requirements.txt

```diff
@@ -1,6 +1,6 @@
 kserve==0.9.0
 torch>=1.12.1
 diffusers==0.11.1
-transformers==4.21.1
+transformers==4.26.1
```

Comment:
When running the inference service with the current sd-inference container I get the following error upon start up:
```
Defaulted container "kfserving-container" out of: kfserving-container, queue-proxy, storage-initializer (init)
/usr/lib/python3/dist-packages/requests/__init__.py:89: RequestsDependencyWarning: urllib3 (1.26.14) or chardet (3.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
[I 230213 18:19:14 service:60] Model Name: sd-test-3
[I 230213 18:19:14 service:61] Model ID: /mnt/pvc/finetunes/sd-test-3
[I 230213 18:19:14 service:122] Loading sd-test-3
Cannot initialize model with low cpu memory usage because `accelerate` was not found in the environment. Defaulting to `low_cpu_mem_usage=False`. It is strongly recommended to install `accelerate` for faster and less memory-intense model loading. You can do so with: 
'''
pip install accelerate
'''
.
Traceback (most recent call last):
  File "/app/service.py", line 185, in <module>
    model.load()
  File "/app/service.py", line 127, in load
    self.load_diffusers()
  File "/app/service.py", line 85, in load_diffusers
    self.pipeline = StableDiffusionPipeline.from_pretrained(
  File "/usr/local/lib/python3.8/dist-packages/diffusers/pipeline_utils.py", line 659, in from_pretrained
    class_obj = getattr(library, class_name)
  File "/usr/local/lib/python3.8/dist-packages/transformers/utils/import_utils.py", line 995, in __getattr__
    raise AttributeError(f"module {self.__name__} has no attribute {name}")
AttributeError: module transformers has no attribute CLIPImageProcessor
```

[This has been fixed in a newer version](https://github.com/huggingface/transformers/issues/20122). We will need to rebuild the ml-containers/sd-finetuner image then update the Argo workflow to use it.